src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -272,13 +271,14 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
         key: 'isAdmin',
         name: translateMessage('Admin consent required'),
         fieldName: 'isAdmin',
-        minWidth:  150,
-        maxWidth:  200,
+        minWidth:  180,
         ariaLabel: translateMessage('Administrator permission'),
         isMultiline: true,
         headerClassName: 'permissionHeader',
         styles: columnCellStyles,
-        onRenderHeader: () => renderColumnHeader('Admin consent required')
+        onRenderHeader: () => renderColumnHeader('Admin consent required'),
```

Comment:
Should this be translated?

Reply:
Yes. They are translated in the renderColumnHeader implementation